src/events.zig

```diff
@@ -439,6 +442,48 @@ pub const Key = struct {
     repeat: bool,
 };
 
+/// Mouse button event structure
+///
+/// ## Version
+/// This struct is available since SDL 3.2.0.
+///
+/// ## Code Examples
+/// ```zig
+///                 see https://wiki.libsdl.org/SDL3/SDL_MouseButtonFlags
+///                                                              left button
+/// if (event.* == .mouse_button_down and event.mouse_button_down.button == 1) {
+///     pressing = true;
+/// }
+///                                                              left button
+/// if (event.* == .mouse_button_up and event.mouse_button_up.button == 1) {
+///     pressing = false;
+/// }
+/// if (pressing) {
+///     std.debug.print("pressing\n", .{});
+/// } else {
+///     std.debug.print("not pressing\n", .{});
+/// }
+/// ```
+pub const MouseButton = struct {
+    /// Common event information.
+    common: Common,
+    /// The window with mouse focus, if any.
+    window_id: ?video.WindowID = null,
+    /// The mouse instance id in relative mode, or null.
+    which: ?mouse.ID,
+    /// The mouse button index.
+    button: u8,
+    /// true if the button is pressed.
+    down: bool,
+    /// 1 for single-click, 2 for double-click, etc.
+    clicks: u8,
+    padding: u8,
```

Comment:
Padding is unused for the wrapped event type and can be removed. It is only necessary for matching C structures in memory exactly, which is not needed here.